README.md

```diff
@@ -1,101 +1,81 @@
-1. Remove the default remote: `git remote rm origin` (notice that `git remote -v` not gives you back nothing)
-1. Create a new repo on GitHub with the name of `[what you want to name the repo]` to be consistent with naming
-1. Copy the address that you would use to clone down this repo - something like `git@github.com:...`
-1. Add this remote to your cloned down repo: `git remote add origin [address you copied in the previous step]` - do not include the brackets
+2. hit this green box that says `Code`
 
-Now try to commit something (just add a line in the README) and push it up to your new repo. If everything is setup correctly, you should see the changes on GitHub.
+![green box]()
 
-## Setup
+3. Copy the SSH
 
-After one person has gone through the steps of cloning down this repo and editing the remote, everyone should clone down the repo.
+![Copy SSH]()
 
-Then install the library dependencies. Run:
+4. Open the terminal
+5. Type `git clone` + paste the SSH link
 
-```bash
-npm install
-```
 
-To verify that it is setup correctly, run `npm start` in your terminal. Go to `http://localhost:8080/` and you should see a page with the Turing logo image and a beautiful gradient background. If that's the case, you're good to go. Enter `control + c` in your terminal to stop the server at any time.
+### **To Run the Application and View the Webpage**
 
-## Where to Add Your Code
+1. In the main-directory, run `npm start`
 
-### JavaScript
+You should see a similar screen below.
 
-You have to be very intentional with where you add your feature code. This repo uses a tool called [webpack](https://webpack.js.org/) to combine many JavaScript files into one big file. Webpack enables you to have many, separate JavaScript files to keep your code organized and readable. Webpack expects all of your code files to be in a specific place, or else it doesn't know how to combine them all behind the scenes.
+![View Website](https://user-images.githubusercontent.com/89413678/161453492-049643ab-135d-4d7a-a86a-7802d8468ac8.png)
 
-**Create all of your feature code files in the `src` directory.**
+2. To view the webpage, copy and paste the first link in your console under `webpack server` into the input field in the window.
 
-Since code is separated into multiple files, you need to use the `import` and `export` syntax to share code across file.
+![View Website](https://user-images.githubusercontent.com/89413678/161453609-03cf671a-359b-4668-af50-85df2e7f1f25.png)
 
-Here is a video that walks through some information about [import and export](https://www.youtube.com/watch?v=_3oSWwapPKQ). There are a lot of resources out there about `import` and `export`, and resources will sometimes call them `ES6 modules`. It's something you will see in React and beyond.
 
-### HTML
+![Apr-18-2022 17-27-32](https://user-images.githubusercontent.com/89413678/163892630-60d1c7bb-6f59-47cd-92b7-3c46845be6e2.gif)
```

Comment:
When I went to the above link to check out the gif, it showed me a screen from the "whats cookin" project?